pyproject.toml

```diff
@@ -2,5 +2,121 @@
 requires = ["setuptools>=64", "setuptools_scm[toml]>=6.2", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+description = "One line description of your module"
+dependencies = []
+dynamic = ["version"]
+license = { file = "LICENSE" }
+name = "python3-pip-skeleton"
+readme = "README.rst"
```

Comment:
Just checking we don't need `file =` here?

Reply:
We could use `license.file = "LICENSE"` instead, which seems neater to me so I've updated it

Reply:
Never mind, just realised what you were actually asking. No, the readme always points to a file and can never be an in place string so has no such option.